docs/guides/advanced/query_api.md

```diff
@@ -197,22 +194,21 @@ else:
 
 ### Additional Configuration
 
-The Logfire API supports various query parameters and response formats to give you flexibility in how you retrieve your data:
+The Logfire API supports various response formats and query parameters to give you flexibility in how you retrieve your data:
 
 - **Response Format**: Use the `Accept` header to specify the response format. Supported values include:
```

Comment:
I don't think multiple levels of bullet point work in mkdocs by default.

Reply:
It seems to be working: https://4d3d712c.logfire-docs.pages.dev/guides/advanced/query_api/#additional-configuration